src/supervisor.c

```diff
@@ -133,6 +133,12 @@ void _svNextByte() {
 	SPI1->DR = spiBufferTX[idx];
 }
 
+// standaloneModeEnable - enable standalone operation
+void standaloneModeEnable() {
+	// set flag in outgoing SPI buffer to enable standlone mode
+	SV_OUT->flags |= 1;

```

Comment:
Are we aware currently of what the remaining bits of this `flags` byte (I assume it to be `uint8_t`) do?